go.mod

```diff
@@ -15,11 +15,18 @@ require (
 	github.com/hashicorp/hcl/v2 v2.9.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.0
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/onsi/ginkgo v1.8.0 // indirect
```

Comment:
please remove any changes to go.mod/go.sum files, as they are not needed.

Reply:
removed.